runtime/onert/core/include/compiler/Compiler.h

```diff
@@ -133,7 +142,7 @@ class Compiler
   // have to add allocate non-constant tensor memory of executors in execution time when each
   // subgraph is called.
   State _state;
-  CompilerOptions _options;
+  CompilerOptions &_options;
```

Comment:
Why did you change this to a reference member?

Reply:
@ragmani `nnfw_set_available_backends()`, `nnfw_set_config()` and `nnfw_get_config()` reads and modifies the compiler option of `nnfw_session`. `CompilerOptions` should be a reference to `nnfw_session`'s `CompilerOptions`.

Reply:
How about change it a pointer member instead?

Reply:
I used reference because
1) I don't want null check which is necessary for pointer.
2) Using pointer type requires more changes.

Why do you prefer pointer?

Reply:
It is nothing. Ignore it.